README.md

```diff
@@ -668,7 +668,7 @@ To learn more about Firebase Test Lab and Google Cloud SDK, please go and visit
 
   ```console
   cat $ANDROID_SDK_ROOT/tools/source.properties | grep Pkg.Revision
-  cat $ANDROID_SDK_ROOT/cmdline-tools/tools/source.properties | grep Pkg.Revision
+  cat $ANDROID_SDK_ROOT/cmdline-tools/cmdline-tools/source.properties | grep Pkg.Revision
```

Comment:
It seems like Google has corrected its behavior, now it can create the proper `cmdline-tools` directory, thus we don't have to. Please remove the redundant `cmdline-tools` path, thanks.